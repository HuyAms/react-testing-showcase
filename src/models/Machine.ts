export enum Size {
	_size1 = 'size1',
	_size2 = 'size2',
	_size4 = 'size4',
}
