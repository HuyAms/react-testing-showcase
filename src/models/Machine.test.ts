import {Size} from './Machine'

describe('Machine', () => {
	it('should have a correct Size enum', () => {
		expect(Size).toMatchSnapshot()
	})
})
