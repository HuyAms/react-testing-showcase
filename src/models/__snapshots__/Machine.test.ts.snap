// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Machine should have a correct Size enum 1`] = `
Object {
  "_size1": "size1",
  "_size2": "size2",
  "_size4": "size4",
}
`;
